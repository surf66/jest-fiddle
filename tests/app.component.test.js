import React from 'react';
import App from '../src/js/components/app.component.js';

test('App component snapshot test', () => {
  const tree = shallow(<App />);
  
  expect(tree).toMatchSnapshot();
});

test('should update notes when _onSubmit is called', () => {
  const component = shallow(<App />).instance();
  
  component.setState({ notes: [] });
  component._onSubmit('new note');

  expect(component.state.notes[0]).toEqual('new note');
});