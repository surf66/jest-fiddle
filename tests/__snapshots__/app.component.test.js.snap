// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component snapshot test 1`] = `
<div>
  <p>
    Notes
  </p>
  <NoteList
    notes={
      Array [
        "One",
        "Two",
        "Three",
      ]
    }
  />
  <Form
    onSubmit={[Function]}
  />
</div>
`;
